frontend/testing/cypress/src/selectors/dashboard.js

```diff
@@ -4,17 +4,21 @@ import * as testids from '../../../testids';
 const getAllAppsHeader = () => cy.findByRole('heading', { name: texts['dashboard.all_apps'] });
 const getUserAppsListHeader = () => cy.findByRole('heading', { name: texts['dashboard.my_apps'] });
 const getFavouritesHeader = () => cy.findByRole('heading', { name: texts['dashboard.favourites'] });
-const getSearchResultsHeader = () => cy.findByRole('heading', { name: texts['dashboard.search_result'] });
-const getOrgAppsHeader = (org) => cy.findByRole('heading', { name: texts['dashboard.org_apps'].replace('{{orgName}}', org) });
+const getSearchResultsHeader = () =>
+  cy.findByRole('heading', { name: texts['dashboard.search_result'] });
+const getOrgAppsHeader = (org) =>
+  cy.findByRole('heading', { name: texts['dashboard.org_apps'].replace('{{orgName}}', org) });
 const getUserAppsList = () => getUserAppsListHeader().next();
 const getSearchResults = () => getSearchResultsHeader().next();
-const getLinksCellForApp = (table, name) => table.findByRole('cell', { name }).siblings('div[data-field=\'links\']');
+const getLinksCellForApp = (table, name) =>
+  table.findAllByRole('cell', { name }).siblings("div[data-field='links']");
```

Comment:
Her endret den til `findAllByRole` fordi cypress testene feilet i to steder dashboard og wcag med samme feil og det er " Fant flere elementer med rollen "celle" og navnet "auto-designer-app-lokal" ".

Reply:
Det kan se ut som at det har blitt gjort en endring som gjør at den samme appen blir opprettet både for Testdepartementet og for testbrukeren. Vi bør nok heller finne ut hvorfor det skjer.